### ISSUE TITLE
The detector is normal, but id predictor gives a big loss.

### ISSUE THREAD

**Issue by LuletterSoul** [MCG-NJU/MOTIP#37]
Hi, I want to try other detectors based on the MOTIP framework. After training with the DanceTrack and Crowdhuman datasets, I found that the detector results were normal, but the ID predictions were all wrong. It is found that id_decoder loss is greater than 1.5x. According to the log you provided, id_decoder loss can be converged to 0.2x.

```
[Epoch: 10] [800/10518] [tps: 2.86s] overall_id_loss = 15.4230 (15.7277); id_loss = 1.5423 (1.5728); learning_rate = 0.0000 (0.0000); detr_grad_norm = 5.5492 (6.5566); other_grad_norm = 3.7698 (3.8111); 
```

Would you like to ask if there is a special optimization strategy for ID Decoder, or am I using it wrong?

**Comment by HELLORPG** [comment]
May I ask whether you are training with our [given configuration](https://github.com/MCG-NJU/MOTIP/blob/main/configs/r50_deformable_detr_motip_dancetrack_trainval_joint_ch.yaml) and following our documents?

**Comment by LuletterSoul** [comment]
@HELLORPG Thank you for your patient reply. I have found the corresponding problem now. I made a mistake with the output of the detector.
